assets/runtime/config/gitlab-pages/config

```diff
@@ -4,3 +4,5 @@ auth-redirect-uri={{GITLAB_PAGES_ACCESS_REDIRECT_URI}}
 auth-secret={{GITLAB_PAGES_ACCESS_SECRET}}
 gitlab-server={{GITLAB_PAGES_ACCESS_CONTROL_SERVER}}
 artifacts-server={{GITLAB_PAGES_ARTIFACTS_SERVER_URL}}
+internal-gitlab-server=http://localhost:8181
+api-secret-key=/home/git/gitlab/.gitlab_pages_secret
```

Comment:
you should use GITLAB_INSTALL_DIR here too match config in assets/runtime/functions

Reply:
@achretien Fixed at 73dae28